src/steps/create-lead.ts

```diff
@@ -39,10 +41,13 @@ export class CreateLead extends BaseStep implements StepInterface {
 
   async executeStep(step: Step): Promise<RunStepResponse> {
     const stepData: any = step.getData().toJavaScript();
-
+    const dateTokenFormat = /\d{4}-\d{2}-\d{2}(?:.?\d{2}:\d{2}:\d{2})?/;
```

Comment:
- [ ] We might also want to update this with a prefixed `^` and an affixed `$`.  Otherwise, a free-text string that happened to contain a date would be matched and attempt to be converted:

![Screen Shot 2020-06-23 at 5 42 51 PM](https://user-images.githubusercontent.com/3496491/85483203-3ef2ef00-b579-11ea-993b-ed6790046d53.png)
\d{2}:\d{2}:\d{2})?$`

Recommend: `^\d{4}-\d{2}-\d{2}(?:.?